### DISCUSSION TITLE
How often do weed volume, weed filler, etc. update parameter changes on weed master ?

### DISCUSSION THREAD

**Discussion by aoberest** [seaweedfs/seaweedfs#2029] [Q&A] [answered]
Hello

How often do weed volume, weed filler, etc. update parameter changes on weed master ?
or does the request for parameters occur only at startup ?

For example
```
weed master -h
-metrics.address string
Prometheus gateway address <host>:<port>
```

**Comment by chrislusf** [comment, marked as answer]
Only at startup.

**Reply by aoberest** [reply]
Maybe if it is possible to do periodic parameter polling and application without restarting ?

**Reply by chrislusf** [reply]
metrics address should not be changed often, right?

**Reply by aoberest** [reply]
Yes, I agree.

**Reply by hamidreza-hosseini** [reply]
> metrics address should not be changed often, right?

@chrislusf 
Yes the metric address is fixed, but other parameters like "-metrics.intervalSeconds" maybe need to change sometimes for debugging purpose
It would be good If weed do it periodiclly
